demo.html

```diff
@@ -294,5 +349,6 @@ <h1 class="section-title">Buttons</h1>
      </code>
    </div>
  </header>
+ </div>
 </body>
 </html>
```

Comment:
With all of this code, I tended to get lost with where it began and where it ended. A little bit more formatting by adding space above and below then making sure things are indented correctly goes  a long way in terms of readability.